src/widgets/allergies/allergy-form.component.tsx

```diff
@@ -198,50 +184,53 @@ export default function AllergyForm(props: AllergyFormProps) {
       props.match.params,
       abortController
     ).then(response => {
-      response.status == 200 && navigate();
+      response.status === 200 && navigate();
     }, createErrorHandler);
     return () => abortController.abort();
   };
 
-  const handleCreateFormSubmit = event => {
-    event.preventDefault();
-    const patientAllergy: Allergy = {
-      allergenType: allergenType,
-      codedAllergenUuid: codedAllergenUuid,
-      severityUuid: reactionSeverityUuid,
-      comment: comment,
-      reactionsUuid: patientAlleryReaction
-    };
-    const abortController = new AbortController();
-    savePatientAllergy(patientAllergy, patientUuid, abortController)
-      .then(response => {
-        response.status == 201 && navigate();
-      })
-      .catch(createErrorHandler());
-    return () => abortController.abort();
-  };
-
   function navigate() {
     history.push(`/patient/${patientUuid}/chart/allergies`);
     props.closeComponent();
   }
 
   const setCheckedValue = uuid => {
```

Comment:
I know you didn't name it, but this is a very bad function name. It's disguised as a setter and obfuscates that it is all about allergy reactions. I suggest something like `reactionIsPresent` or `allergyHasReaction`